### PR TITLE
Lab -1 updates.

### PR DESCRIPTION
**Pull request by nagi9**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/beginner/120_network-policies/tigera/environment.md [hunk 1]
```diff
@@ -1,43 +1,22 @@
...
+   - How to use Policy Tiers and Network Sets to define network policies that define the “guard rails” for your entire Kubernetes platform
+   - How to use DNS policy rules to enable external access for applications in specific namespaces
+   - How to use the Flow Visualizer to gain visibility into existing traffic flows within the cluster and select workloads to recommend policies using Policy Recommendations
+   - How to use Policy Impact Preview and Staged Network Policies to evaluate the impact of new network policies and changes to existing policies before they are enforced within the cluster
 
-The $CLUSTER_NAME variable is the same that you used to create the cluster using the 'eksctl' command at the beginning of the workshop.  If you followed the directions, it will be 'eksworkshop-eksctl'
+### Lab Source : [Policy Automation and External Access](https://info.tigera.io/rs/805-GFH-732/images/Calico-Enterprise-Lab01.pdf?mkt_tok=eyJpIjoiTVRjek1EWTNNemRtTkRVdyIsInQiOiJoY3l5aUE1MEFiRk1Ia2NLSDN6b0JWRU5HMlhORUswTm14MldkQ1owbkl4ZkN6Vm5LRmdXOGZ1R094MG5KOFVYXC9GcmdOUmY4YzdjbUo1dGdaNUFMQ1E9PSJ9)
```
Comment (brentley) [inline content/beginner/120_network-policies/tigera/environment.md:21]: Help me understand what this external PDF is about?

Comment (nagi9) [reply content/beginner/120_network-policies/tigera/environment.md:21]: It's link posted on our website for the step by step walkthrough about the lab.

Comment (brentley) [reply content/beginner/120_network-policies/tigera/environment.md:21]: Can we get the step-by-step put into the workshop instead (like it is now)? We don't want external links out to be part of the main "substance" of the workshop. 

If you'd rather send folks to your site to explore, we can delete the section on Enterprise, and just link to the enterprise high-level info in a "for more information, click here" style reference.

Comment (nagi9) [reply content/beginner/120_network-policies/tigera/environment.md:21]: Ok sure, let me work on that and will submit PR's for the step by step process but it's going to take some time.

I can work on that next week.

**Comment by brentley** [comment]
sounds good. thanks!
GitHub <notifications@github.com> wrote:
“@nagi9 commented on this pull request.”


“In content/beginner/120_network-policies/tigera/environment.md:”

>  
-The $CLUSTER_NAME variable is the same that you used to create the cluster using the 'eksctl' command at the beginning of the workshop.  If you followed the directions, it will be 'eksworkshop-eksctl'
+### Lab Source : [Policy Automation and External Access](https://info.tigera.io/rs/805-GFH-732/images/Calico-Enterprise-Lab01.pdf?mkt_tok=eyJpIjoiTVRjek1EWTNNemRtTkRVdyIsInQiOiJoY3l5aUE1MEFiRk1Ia2NLSDN6b0JWRU5HMlhORUswTm14MldkQ1owbkl4ZkN6Vm5LRmdXOGZ1R094MG5KOFVYXC9GcmdOUmY4YzdjbUo1dGdaNUFMQ1E9PSJ9)
“Ok sure, let me work on that and will submit PR's for the step by step process but it's going to take some time.

I can work on that next week.”

**Pull request closed without merging**